### ISSUE TITLE
Empty administrators page after upgrade from 8.0.0-22.1 to 8.0.0-23.1

### ISSUE THREAD

**Issue by tps800** [owncloud/core#14436]
### Steps to reproduce
1. Install OwnCloud, Configure OwnCloud, Configure LDAP access
2. Upload a wealth of pictures
3. Browse to your owncloud instance
4. login with "Administrator"
5. Switch to "Administration"

### Expected behaviour
Show administration tools

### Actual behaviour
After page loaded page stays empty

### Server configuration
**Operating system:** Oracle Linux 6.6

**Web server:** Apache 2.2.15-39.0.1

**Database:** PostgreSQL 9.3.6

**PHP version:** PHP 5.5.21

**ownCloud version:** 8.0.0-23.1

**Updated from an older ownCloud or fresh install:** Upgrade from 6.5

**List of activated apps:** ?

**The content of config/config.php:** 

```
<?php
$CONFIG = array (
  'instanceid' => 'oc692ddacb52',
  'passwordsalt' => '',
  'trusted_domains' =>
  array (
    0 => '<server>.<local>.<domain>',
  ),
  'datadirectory' => '/data/owncloud/data',
  'dbtype' => 'pgsql',
  'version' => '7.0.1.1',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'owncld',
  'dbpassword' => '',
  'installed' => true,
  'ldapIgnoreNamingRules' => false,
  'theme' => '',
  'maintenance' => false,
  'loglevel' => '0',
  'mail_smtpmode' => 'php',
  'mail_smtpname' => 'admin',
  'mail_smtppassword' => '',
);

```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

### Client configuration
**Browser:**
* Windows XP/7/8:
 * Chrome (stable, beta, snapshot)
 * Chromium (stable, continuous)
 * Firefox (stable, nightly)
 * Opera (stable)
 * lynx (from cygwin)
 * w3m (from cygwin)
* Linux (Debian, Ubuntu 12.04/14.04/14.10, Fedora 19/20/21, Oracle Linux 6.5/7.0)
 * Chrome (stable, beta, snapshot)
 * Chromium (stable, continuous)
 * Firefox (stable, nightly)
 * Opera (last available)
 * Konqueror (stable)
 * lynx (stable)
 * w3m (stable)

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by RealRancor** [comment]
Hi,

please provide the installed and activated apps (see occ command line tool) and the entries of your data/owncloud.log

**Comment by tps800** [comment]
# sudo -u apache ./occ app:list
Enabled:
 - activity
 - files
 - files_pdfviewer
 - files_sharing
 - files_trashbin
 - gallery
 - imprint
 - ocusagecharts
 - password_policy
 - user_ldap
Disabled:
 - external
 - files_encryption
 - files_external
 - files_locking
 - files_texteditor
 - files_versions
 - files_videoviewer
 - firstrunwizard
 - provisioning_api
 - templateeditor
 - updater
 - user_external
 - user_webdavauth

**Comment by RealRancor** [comment]
Hi,

try to disable (also via occ)

imprint
ocusagecharts
password_policy

to see if any of those 3rdparty apps is causing this.

**Comment by tps800** [comment]
```
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => \n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"readAttribute: uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de found","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => uid\n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => primaryGroupID\n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"f97c3b0a92b429ce85fe4848fd87103b","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Requested attribute primarygroupid not found for uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de","level":0,"time":"2015-02-24T09:04:22+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => \n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"readAttribute: uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de found","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => uid\n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"initializing paged search for  Filter objectClass=* base Array\n(\n    [0] => uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de\n)\n attr Array\n(\n    [0] => primaryGroupID\n)\n limit 500 offset 0","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Ready for a paged search","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"817f1278253ec4378396ed1f3959bf82","remoteAddr":"10.160.2.12","app":"user_ldap","message":"Requested attribute primarygroupid not found for uid=sct-muc,ou=users,ou=z,ou=muc,dc=bfs,dc=de","level":0,"time":"2015-02-24T09:14:40+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/"}
{"reqId":"523d2d229de1de4beb35e597c3b00fd7","remoteAddr":"10.160.2.12","app":"webdav","message":"Exception: {\"Message\":\"No basic authentication headers were found\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/connector\\\/sabre\\\/auth.php(123): Sabre\\\\DAV\\\\Auth\\\\Backend\\\\AbstractBasic->authenticate(Object(OC_Connector_Sabre_Server), 'ownCloud')\\n#1 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/connector\\\/sabre\\\/auth.php(103): OC_Connector_Sabre_Auth->auth(Object(OC_Connector_Sabre_Server), 'ownCloud')\\n#2 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/Sabre\\\/DAV\\\/Auth\\\/Plugin.php(108): OC_Connector_Sabre_Auth->authenticate(Object(OC_Connector_Sabre_Server), 'ownCloud')\\n#3 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod('PROPFIND', '')\\n#4 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/Sabre\\\/DAV\\\/Server.php(433): call_user_func_array(Array, Array)\\n#5 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/Sabre\\\/DAV\\\/Server.php(455): Sabre\\\\DAV\\\\Server->broadcastEvent('beforeMethod', Array)\\n#6 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/Sabre\\\/DAV\\\/Server.php(214): Sabre\\\\DAV\\\\Server->invokeMethod('PROPFIND', '')\\n#7 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files\\\/appinfo\\\/remote.php(61): Sabre\\\\DAV\\\\Server->exec()\\n#8 \\\/var\\\/www\\\/html\\\/owncloud\\\/remote.php(54): require_once('\\\/var\\\/www\\\/html\\\/o...')\\n#9 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/Sabre\\\/DAV\\\/Auth\\\/Backend\\\/AbstractBasic.php\",\"Line\":73}","level":0,"time":"2015-02-24T09:17:10+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav"}
{"reqId":"197a59e7619e762f19483032d4a6c847","remoteAddr":"10.160.2.12","app":"core","message":"OC\\Tags::__construct, tags: Array\n(\n)\n","level":0,"time":"2015-02-24T09:49:08+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/files\/ajax\/list.php?dir=%2F&sort=name&sortdirection=asc"}
{"reqId":"9aa62d1d9fb22d1089cf73d1a9df1039","remoteAddr":"10.160.2.12","app":"index","message":"Exception: {\"Message\":\"js file not found: script:js\\\/chosen\\\/chosen.jquery.min serverroot:\\\/var\\\/www\\\/html\\\/owncloud\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/templatelayout.php(155): OC\\\\Template\\\\ResourceLocator->find(Array)\\n#1 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/templatelayout.php(106): OC_TemplateLayout::findJavascriptFiles(Array)\\n#2 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/template.php(118): OC_TemplateLayout->__construct('user', 'settings')\\n#3 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/template\\\/base.php(103): OC_Template->fetchPage()\\n#4 \\\/var\\\/www\\\/html\\\/owncloud\\\/settings\\\/admin.php(127): OC\\\\Template\\\\Base->printPage()\\n#5 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/route.php(135) : runtime-created function(1): require_once('\\\/var\\\/www\\\/html\\\/o...')\\n#6 [internal function]: __lambda_func(Array)\\n#7 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/router.php(250): call_user_func('\\\\x00lambda_248', Array)\\n#8 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/base.php(763): OC\\\\Route\\\\Router->match('\\\/settings\\\/admin')\\n#9 \\\/var\\\/www\\\/html\\\/owncloud\\\/index.php(36): OC::handleRequest()\\n#10 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/template\\\/resourcelocator.php\",\"Line\":46}","level":4,"time":"2015-02-24T09:49:14+00:00","method":"GET","url":"\/owncloud\/index.php\/settings\/admin"}
{"reqId":"9aa62d1d9fb22d1089cf73d1a9df1039","remoteAddr":"10.160.2.12","app":"PHP","message":"js file not found: script:js\/chosen\/chosen.jquery.min serverroot:\/var\/www\/html\/owncloud at \/var\/www\/html\/owncloud\/lib\/private\/template\/resourcelocator.php#46","level":3,"time":"2015-02-24T09:49:14+00:00","method":"GET","url":"\/owncloud\/index.php\/settings\/admin"}
```

**Comment by tps800** [comment]
Imprint was the bad one. After disabling all is fine again with the admin page.

**Comment by RealRancor** [comment]
Ok, good to know. Then please report this issue to the issue tracker of the imprint app.

**Comment by PVince81** [comment]
Taking care of this now.

**Comment by PVince81** [comment]
Fix is here: https://github.com/owncloud/apps/pull/2020

**Comment by tps800** [comment]
Seems the fix works partly -- it stops apache but does not restart it. After going up to 8.0.0-24-1 apache is stopped, maintenance mode is ended, but apache never started again!

**Comment by DeepDiver1975** [comment]
> Fix is here: owncloud/apps#2020

merged

**Comment by nickvergessen** [comment]
Closing this one then?
